LiteNetLibSystemExample/core/GameManager.cs

```diff
@@ -37,6 +38,13 @@ public GameManager()
     // Called when the node enters the scene tree for the first time.
     public override void _Ready()
     {
+        //check for args to override server settings from command line args
+        var args = OS.GetCmdlineArgs();
+        if (args.Contains("--server"))
+            IsServer = true;
+        if (args.Contains("--client"))
+            IsServer = false;
```

Comment:
We always use curly braces for code blocks to avoid unnecessary confusion and sloppiness. This practice helps us avoid trapping ourselves or other colleagues. I adopted this habit from my daily work with C++.

```suggestion
        if (args.Contains("--server"))
        {
            IsServer = true;
        }
        if (args.Contains("--client"))
        {
            IsServer = false;
        }
```